### PR TITLE
edge 1018 - improve some descriptions

### PR DESCRIPTION
**Pull request by smckinnon-bw**
small changes to description that I left out of the last PR 'cause I was too hasty.  Sorry for the spam - hopefully I'm learning

### CODE REVIEW

File: site/docs/webrtc/webhooks.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ import TabItem from "@theme/TabItem";
...
 
 ## Life Cycle Events
 
 The events that will trigger invocation of the specified URL are:
 
-- 'onConnect' - establishment of the websocket connection that will allow the flow of WebRTC Media, an explicit action on the part of the websocket client.
-- 'onDisconnect' - explicit removal of the websocket connection, prompted by explicit server or client action.
-- 'onLeave' - loss of the websocket connection, possibly due to loss of IP connectivity.
+- `onConnect` - an explicit action on the part of the client that establishes a signalling connection to control the flow of WebRTC Media.
+- `onDisconnect` - explicit removal of the signalling connection, prompted by explicit server or client action.
```
Comment (carrington-bw) [inline site/docs/webrtc/webhooks.md:36]: I feel like the descriptions you have for onDisconnect and onLeave should be reversed (i.e. the onLeave seems to actually describe onDisconnect and vice versa)

**Review by hamermike — CHANGES_REQUESTED** [review]
Within the SDKs - Web and mobile we use "signaling". Valid spelling seems to be either though.

File: site/docs/webrtc/webhooks.md (hunk 1 above)
Comment (carrington-bw) [reply site/docs/webrtc/webhooks.md:36]: Something else to think about and consider is what information an application developer needs in order to write their application, and tailoring the content to what an application developer is looking for and needs to know.

File: site/docs/webrtc/webhooks.md [hunk 2]
```diff
@@ -18,23 +18,23 @@ import TabItem from "@theme/TabItem";
...
 :::
 
 ## Life Cycle Events
 
 The events that will trigger invocation of the specified URL are:
 
-- 'onConnect' - establishment of the websocket connection that will allow the flow of WebRTC Media, an explicit action on the part of the websocket client.
-- 'onDisconnect' - explicit removal of the websocket connection, prompted by explicit server or client action.
-- 'onLeave' - loss of the websocket connection, possibly due to loss of IP connectivity.
+- `onConnect` - an explicit action on the part of the client that establishes a signalling connection to control the flow of WebRTC Media.
```
[the remark this answers is not in the document]
Comment (smckinnon-bw) [reply site/docs/webrtc/webhooks.md:35]: it's perfectly good Canadian spelling :-)

**Review by ajrice6713 — APPROVED** [review]

**Review by hamermike — APPROVED** [review]

**Pull request merged**